### ISSUE TITLE
Different scaling

### ISSUE THREAD

**Issue by sigmoidx** [MarkPrecursor/EEDS-keras#2]
Hello?

Thanks for upload. And I wonder the weight is built on an assumption that the scale is fixed at 2. I wonder changing the scale parameter means that I may have to retrain it with my custom data. 

Thanks,

**Comment by MaokeAI** [comment]
Yes, this weight only trained for the scale of 2. If you want to upscale for another scale, the model need to be changed a little (the deconv part), and retrain the model

**Comment by sigmoidx** [comment]
I have changed every scale parameter on EEDS, EED, EES and ran python EEDS.py

But there was an error. I used an input size of 16x16 and expect a 64x64 sized output with a scale of 4.

`Caused by op u'sub', defined at:
  File "EEDS.py", line 97, in <module>
    EEDS_train()
  File "EEDS.py", line 41, in EEDS_train
    _EEDS = model_EEDS()
  File "EEDS.py", line 36, in model_EEDS
    model.compile(optimizer=Adam, loss='mse')
  File "/home/marsha/anaconda2/lib/python2.7/site-packages/keras/engine/training.py", line 911, in compile
    sample_weight, mask)
  File "/home/marsha/anaconda2/lib/python2.7/site-packages/keras/engine/training.py", line 436, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/marsha/anaconda2/lib/python2.7/site-packages/keras/losses.py", line 8, in mean_squared_error
    return K.mean(K.square(y_pred - y_true), axis=-1)
  File "/home/marsha/anaconda2/lib/python2.7/site-packages/tensorflow/python/ops/math_ops.py", line 838, in binary_op_wrapper
    return func(x, y, name=name)
  File "/home/marsha/anaconda2/lib/python2.7/site-packages/tensorflow/python/ops/gen_math_ops.py", line 2501, in _sub
    result = _op_def_lib.apply_op("Sub", x=x, y=y, name=name)
  File "/home/marsha/anaconda2/lib/python2.7/site-packages/tensorflow/python/framework/op_def_library.py", line 767, in apply_op
    op_def=op_def)
  File "/home/marsha/anaconda2/lib/python2.7/site-packages/tensorflow/python/framework/ops.py", line 2506, in create_op
    original_op=self._default_original_op, op_def=op_def)
  File "/home/marsha/anaconda2/lib/python2.7/site-packages/tensorflow/python/framework/ops.py", line 1269, in __init__
    self._traceback = _extract_stack()

InvalidArgumentError (see above for traceback): Incompatible shapes: [64,32,32,1] vs. [64,64,64,1]
`

Thanks,

**Comment by MaokeAI** [comment]
This error may because the wrong output stride. please check if the stride of deconv layer is set to 4 with suitable padding
